springdb/frontend/src/sections/@dashboard/app/AEList.js

```diff
@@ -94,12 +95,17 @@ export default function AEList({target, actions, tableResults, filterHandler}) {
                             <TableCell>
                                 {/*Empty*/}
                             </TableCell>
+                            {displayAE_Weights ?
```

Comment:
i think you can just check for if tableResults is empty here right?

Reply:
@iEatYanYans  Yes, I checked if it's empty. I can get rid of the comment.

Reply:
ah I meant instead of making a new variable, replacing lines 94-109 with this:

`                    <TableHead>
                        {tableResults.length > 0 ?
                            <TableRow>
                                <TableCell>
                                </TableCell>
                                <TableCell>
                                    Adverse Event
                                </TableCell>
                                <TableCell>
                                    Weight
                                </TableCell>
                            </TableRow>
                        : null}
                    </TableHead>`

Reply:
hmm the format got weird but I meant this: 
<img width="430" alt="image" src="https://user-images.githubusercontent.com/21980529/166092211-fffddf2b-62c6-468c-8f15-ca203db867e1.png">